website/ADD_CASE_STUDY.md

```diff
@@ -27,18 +27,21 @@ limitations under the License.
-| `icon`            | Relative path to the company/project logo e.g. "/images/logos/powered-by/company_name.png"              |
-| `category`        | `study` for case studies                                                                                |
-| `cardTitle`       | Case study card title for Apache Beam [case studies](https://beam.apache.org/case-studies/) page        |
-| `cardDescription` | Description for [case studies](https://beam.apache.org/case-studies/) page, usually 30-40 words         |
-| `authorName`      | Case study author                                                                                       |
-| `authorPosition`  | Case study author role                                                                                  |
-| `authorImg`       | Relative path for case study author photo, e.g. "/images/case-study/company/authorImg.png"              |
-| `publishDate`     | Case study publish date for sorting at [case studies](https://beam.apache.org/case-studies/), e.g. `2022-10-14T01:56:00+00:00` |
+| Field                 | Description                                                                                                                    |
+|-----------------------|--------------------------------------------------------------------------------------------------------------------------------|
+| `title`               | Case study title, usually 4-12 words                                                                                           |
+| `name`                | Company or project name                                                                                                        |
+| `icon`                | Relative path to the company/project logo e.g. "/images/logos/powered-by/company_name.png"                                     |
+| `category`            | `study` for case studies                                                                                                       |
+| `cardTitle`           | Case study card title for Apache Beam [case studies](https://beam.apache.org/case-studies/) page                               |
+| `cardDescription`     | Description for [case studies](https://beam.apache.org/case-studies/) page, usually 30-40 words                                |
+| `authorName`          | Case study author                                                                                                              |
+| `secondAuthorName`    | Case study additional author, optional param                                                                                   |
+| `authorPosition`      | Case study author role                                                                                                         |
+| `secondAuthorPosition`| Case study additional author role, optional param                                                                              |
```

Comment:
Similarly - Consider changing the parameter name to coauthrorPosition?

Reply:
changed